skipper_test.go

```diff
@@ -329,22 +335,39 @@ func testServerShutdown(t *testing.T, o *Options, scheme string) {
 
 	time.Sleep(shutdownDelay / 2)
 
-	t.Logf("ongoing request passing in before shutdown")
-	r, err := waitConnGet(testUrl)
-	require.NoError(t, err)
-	require.Equal(t, 200, r.StatusCode)
```

Comment:
Here was the problem: request takes 3s so at this points it is 3.5 seconds after shutdown started. The subsequent request surely fails because servers has completed shutdown after this.